code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -62,12 +62,7 @@ var/global/list/datum/stack_recipe/sandstone_recipes = list ( \
 	icon = 'icons/obj/items.dmi'
 	icon_state = "sandbags"
 	singular_name = "sandbag"
-	force = 5
```

Comment:
Lots of useless code in this file so I took away the things that are defined in sheets.dm for them already. No values were changed.